src/bitcoin-cli.cpp

```diff
@@ -642,7 +642,7 @@ class NetinfoRequestHandler : public BaseRequestHandler
         "  send     Time since last message sent to the peer, in seconds\n"
         "  recv     Time since last message received from the peer, in seconds\n"
         "  txn      Time since last novel transaction received from the peer and accepted into our mempool, in minutes\n"
-        "           \"*\" - the peer requested we not relay transactions to it (relaytxes is false)\n"
+        "           \"*\" - the peer did not request that we relay transactions to it (relaytxes is false)\n"
```

Comment:
I don't think this change is more accurate. `relaytxes` is false only when either a peer explicitly requests tx relay to be disabled (via setting the `fRelay` flag added in BIP 37 to false), or when we decide that an outgoing connection should be block-relay-only, in which case we ignore any request from it to relay transactions. Saying "whether we will relay txs with this peer" might be more accurate though.

Reply:
Thanks, "whether we relay transactions with this peer" SGTM.

Reply:
Updated both the getpeerinfo and the -netinfo help docs.